spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/lock/LockRepository.java

```diff
@@ -41,8 +42,9 @@ public interface LockRepository extends Closeable {
 	/**
 	 * Remove a lock from this repository.
 	 * @param lock the lock to remove.
+	 * @return deleted or not.
 	 */
-	void delete(String lock);
+	boolean delete(String lock);
```

Comment:
This is a breaking change in the public API.
Can we keep an old version of this method as `@Deprecated`?

Reply:
After more thinking even if it is a breaking change from the API perspective, it is still save to call this method without checking its return.
So, having the fact that we still are changing behavior of the `RedisLockRegistry`, I probably will be OK to leave this as is and just mention in the Migration Guide.